### ISSUE TITLE
Using Dev Tools to set media queries

### ISSUE THREAD

**Issue by zannain** [illmatic6514/f1-3-c2p1-colmar-academy#3]
Media queries are a crucial tool for making a web page responsive. I recommend using your developer tools to resize your webpage and seeing how your page looks when you resize it. Chrome Dev Tools has an excellent tool built-in tools that allow you to resize your webpage and when you see inconsistencies it tells you the size of the page at which the inconsistency shows up and this is where media queries are amazing tools. For example:

![image](https://user-images.githubusercontent.com/24375741/35258972-fd42d09e-ffd0-11e7-9220-0f3985b4476f.png)

Knowing this, you could add a media query to fix this issue. Using these tools will ensure a great user experience for users visiting your web page. Its also good practice to test for cross-browser inconsistencies because you never know what browser a user visiting your page is using. Different browsers have their own quirks and becoming familiar with these will ensure no matter what browser a user visiting your webpage is using that they will have a good user experience. A good tool to use is [caniuse.com](https://caniuse.com/) which will tell you if specific browsers support certain CSS properties.

**Comment by illmatic6514** [comment]
Yes I use devtools but I didn't find out how I can use @media queries in devtools. Is it even possible?
